src/components/events/Events.tsx

```diff
@@ -84,10 +84,35 @@ export default function Events(props: EventListProps): JSX.Element {
     });
   }, [locale]);
 
-  useEffect(() => {
-    updateTags();
-    setSize(1);
-  }, [filter, setSize, updateTags]);
+  useEffect(() => {  
```

Comment:
These changes should be removed, they don't look to be relevant for this pull request.

Reply:
fixed